content/en/synthetics/platform/private_locations/_index.md

```diff
@@ -666,7 +666,50 @@ For more information about private locations parameters for admins, see [Configu
 
 You can upload custom root certificates to your private locations to have your API and browser tests perform the SSL handshake using your own `.pem` files.
 
-When spinning up your private location containers, mount the relevant certificate `.pem` files to `/etc/datadog/certs` in the same way you mount your private location configuration file. These certificates are considered trusted CA and are used at test runtime. **Note**: If you combine all your `.pem` files into one file, then the order in which the certificates are placed matters. It is required that the intermediate certificate precedes the root certificate to successfully establish a chain of trust.
+{{< tabs >}}
+{{% tab "Linux container" %}}
+
+When spinning up your private location containers, mount the relevant certificate `.pem` files to `/etc/datadog/certs` in the same way you mount your private location configuration file. These certificates are considered trusted CA and are used at test runtime. 
+
+**Note**: If you combine all your `.pem` files into one file, the sequence of the certificates within the file is important. It is required that the intermediate certificate precedes the root certificate to successfully establish a chain of trust.
```

Comment:
The Windows tab has a note that uses the alert box syntax (`<div class="alert alert-info">`), so consider using it here for consistency? But I see other notes elsewhere on the page that use this format, so maybe doesn't matter so much.

Reply:
great feedback!